doc/doxygen/options.dox.in

```diff
@@ -175,22 +174,37 @@ PREDEFINED             = DOXYGEN=1 \
                          DEBUG=1 \
                          DEAL_II_WARNING(x)= \
                          DEAL_II_USE_MT_POSIX=1 \
+                         DEAL_II_CONSTEXPR=constexpr \
+                         DEAL_II_NAMESPACE_OPEN= \
```

Comment:
did you reorder these define with a specific ordering in mind? Maybe we should sort alpabetically?

Reply:
Yes, it's `DEAL_WITH_*` options (and corresponding configurations) last, everything else before that. I extracted the former ones from `base/config.h.in` and would rather not merge them with the others.